src/index.html

```diff
@@ -361,17 +372,17 @@ <h3>Your Drawing</h3>
       openModal(true);
     }
     function exportFromParsonsUI() {
+      openModal(false);
+    }
+    function switchToParameters() {
+      var overlay = document.getElementById('parsonsSettingsOverlay');
+      overlay.className = overlay.className.replace('parameters', '');
```

Comment:
When switch to parameters parameters class should be added to the overlay element. Maybe need to rename methods/buttons